models/index.js

```diff
@@ -0,0 +1,13 @@
+const User = require('./User');
+const Pet = require('./Pet');
```

Comment:
I believe the require statements on lines 1 and 2 need to be lowercase to match the file name. 
Ex: const User = require('./user');